test/spectral_transform.jl

```diff
@@ -90,6 +93,34 @@ end
     end
 end
 
+@testset "Transform: Individual Legendre polynomials (inexact transforms)" begin
+    @testset for trunc in spectral_resolutions_inexact
```

Comment:
Yeah, really like the idea to split between exact grids and inexact for testing!

Reply:
Glad you like it!